extra-network/netatalk/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=netatalk
+PKGSEC=net
+PKGDES="Apple Filing Protocol server"
+PKGDEP="glibc pkg-config db libgcrypt gcc"
```

Comment:
There are a few missing dependencies.

* `/usr/bin/netatalk`: `libevent`
* `/usr/lib/libatalk.so`: `mariadb` `openldap`
* `/usr/lib/netatalk/uams_dhx2_passwd.so`: `libgcrypt`
* `/usr/bin/afpd`: I'm not sure why but this thing linked against `glib`.

Reply:
Thanks